externalPlugins/EX-PO007-NamingConventions.js

```diff
@@ -66,9 +66,17 @@ var plugin = {
 var onPolicy = function(policy, cb) {
   var displayName = policy.getDisplayName(),
     policyType = policy.getType(),
-    prefixes = policyMetaData[policyType].indications,
     found = false,
-    hadWarn = false;
+    hadWarn = false,
+    policyMeta = policyMetaData[policyType];
+    
+  if (!policyMeta) {
+      // No prefix defined for this policy
+      found=false;
+      hadWarn=false;
+      return;
+  }
+  var prefixes = policyMeta.indications;
 
   prefixes.some(function(prefix) {
     if (displayName.startsWith(prefix)) {
```

Comment:
This code is confusing and inelegant.  Should be updated to use modern conventions. Eg
```
found = prefixes.some( p => displayName.startsWith(p)); 
```